docs/contributing/doc_guide.rst

```diff
@@ -1231,23 +915,346 @@ Narrative documentation guidelines
+  continuous integration testing platforms.
+
+.. _doc-build-prereqs:
+
+Prerequisites
+-------------
+
+To install all Python dependencies required to develop PlasmaPy on your
+local computer, enter the top-level directory of the cloned repository
+and run:
+
+.. code-block:: bash
+
+   pip install -e ".[tests,docs]"
+
+It may also be necessary to install the following software:
+
+* `graphviz <https://graphviz.org/download>`__
+* make_ (not necessary for building the documentation with tox_)
+* `pandoc <https://pandoc.org/installing.html>`__
+
+Building documentation with make
+--------------------------------
+
+If you have make_ installed, then you can build the documentation by
+entering the :file:`docs/` directory and running:
+
+.. code-block:: bash
+
+   make html
+
+Including the :samp:`-j {n}` flag in the ``make`` command will enable a
+parallel build, where :samp:`{n}` is replaced with the number of
+processes or ``auto``. To skip building example notebooks, use ``make
+html-nonb`` instead.
+
+You can access the documentation landing page by opening
+:file:`docs/_build/html/index.html` with your browser of choice.
+
+To remove all files previously generated by ``make``, run:
+
+.. code-block:: bash
+
+   make clean
+
+This command is needed when you make a change to a file that does not
+trigger Sphinx_ to rebuild the file that you altered, for example
+modifying a CSS_ file. Using ``make clean-api`` instead will only remove
+the :wikipedia:`API` portion of the documentation build.
+
+To check that hyperlinks are correct, run:
+
+.. code-block:: bash
+
+   make linkcheck
+
+Building documentation with tox
+-------------------------------
+
+You can use tox_ to locally build the documentation by running:
+
+.. code-block:: bash
+
+   tox -e build_docs
+
+You can access the documentation landing page by opening
+:file:`docs/_build/html/index.html` with your browser of choice.
+
+To pass any `options to sphinx-build`_, put them after ``--``, as in the
+following example:
+
+.. code-block:: bash
+
+   tox -e build_docs -- -j=auto -q
+
+The :bash:`-j=auto` option tells `sphinx-build`_ to build the
+documentation in parallel, with the number of processes being
+automatically determined. The :bash:`-q` flag makes `sphinx-build`_
+print out only warnings and errors, which makes them easier to find and
+debug.
+
+You can alternatively shorten the documentation build by running:
+
+.. code-block:: bash
+
+   tox -e build_docs_no_examples
+
+This command will build the documentation without executing the
+:ref:`example notebooks <example_notebooks>`.
+
+.. tip::
+
+   When writing documentation, please make sure to fix any warnings that
+   arise. To enforce this, the ``build_docs`` tox_ environment will fail
+   after completing the documentation build if there are any warnings.
+
 Troubleshooting
 ===============
 
 This section describes how to fix common documentation errors and
-warnings.
+warnings. 🛠️
```

Comment:
Given how Sphinx error messages can sometimes be rather...obfuscating...I seriously considered using the facepalm emoji instead. 🤦